gson/src/main/java/com/google/gson/internal/bind/DateTimeTypeAdapters.java

```diff
@@ -0,0 +1,163 @@
+/*
+ * Copyright (C) 2011-2016 Google Inc.
```

Comment:
what do you mean? 2011-2016 part?

I thought that was the standard way to include the year a file is written.